### PR TITLE
[INLONG-849][Doc] Add Iceberg extract node documentation

### LINKED ISSUE #849: [Improve][Doc] Add Iceberg extract node documentation
### Description

Add `Iceberg` extract node documentation.

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #849 

### Motivation

Add Iceberg extract node documentation.

### Modifications

Add Iceberg extract node documentation.

### Verifying this change

localhost_3000_docs_next_data_node_extract_node_iceberg

### CODE REVIEW

File: docs/data_node/extract_node/iceberg.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    <version>${siteVariables.inLongVersion}</version>
+</dependency>
+`}
+</code></pre>
+
+## Usage
+
+Before creating the Iceberg task, we need a Flink environment integrated with Hadoop
+
+1. Download `Hadoop` on the host.
```
Comment (EMsnap) [inline docs/data_node/extract_node/iceberg.md:33]: on host machine

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/iceberg.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+```
+CREATE TABLE `iceberg_table_source`(
+    PRIMARY KEY (`_id`) NOT ENFORCED,
+    `_id` STRING,
+    `id` INT,
+    `name` STRING,
+    `age` INT)
+    WITH (
+    'metrics.audit.key' = '13&14',
```
Comment (EMsnap) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/iceberg.md:93]: only need iceberg source key

File: docs/data_node/extract_node/iceberg.md (hunk 1 above)
Comment (haifxu) [reply docs/data_node/extract_node/iceberg.md]: Done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/iceberg.md (hunk 2 above)
Comment (haifxu) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/iceberg.md]: Done.

**Comment by EMsnap** [comment]
@vernedeng  PLAL

File: docs/data_node/extract_node/iceberg.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+</dependency>
+`}
+</code></pre>
+
+## Usage
+
+Before creating the Iceberg task, we need a Flink environment integrated with Hadoop
+
+1. Download `Hadoop` on host machine.
+2. Modify `jobmanager.sh` and `taskmanager.sh` in the host and add `Hadoop` environment variables.
```
Comment (vernedeng) [inline docs/data_node/extract_node/iceberg.md:34]: Better to offer a modified startstcript

**Review by dockerzhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Pull request merged**